app/views/layouts/application.slim

```diff
@@ -12,30 +12,33 @@ html lang="en"
   body.MainLayout
     .MainLayout-header
       header.Header role="banner"
-        .Header-logo
-          .Logo
-            = link_to "Afterwork", root_path, class: 'Logo-link'
-        .Header-nav
-          nav.Navbar role="navigation"
-            ul
-              li.Navbar-item = link_to "Events", events_path, class: 'Navbar-link'
-              - if current_user
-                li.Navbar-item = link_to "#{User.user_name(current_user)}", user_path(current_user), class: 'Navbar-link'
-                li.Navbar-item = link_to "Sign out", destroy_user_session_path, method: :delete, class: 'Navbar-link'
-              -else
-                li.Navbar-item = link_to "Sign in with Twitter", user_omniauth_authorize_path(:twitter), class: 'Navbar-link'
-              li.Navbar-item = link_to "About", root_path, class: 'Navbar-link'
+        .Container
+          .Header-logo
+            .Logo
+              = link_to "Afterwork", root_path, class: 'Logo-link'
+          .Header-nav
+            nav.Navbar role="navigation"
```

Comment:
Why is `.Navbar` attached to `nav`? I'm looking at your markup and it seems to make more sense if it was attached to the `ul`